scripts/add_currency_notes.py

```
@@ -0,0 +1,194 @@
+)
+resp.raise_for_status()
+rows = resp.json()["list"]
+print(f"Fetched {len(rows)} records.")
+
+updated = skipped = 0
+for row in rows:
+    code = row.get("Code", "")
+    if row.get("Notes") or code not in NOTES:
+        skipped += 1
+        continue
+    patch = requests.patch(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)